packages/account/src/account.test.ts

```diff
@@ -221,6 +221,8 @@ describe('Account', () => {
     expect(addResourcesSpy).toHaveBeenCalledWith(resourcesToSpend);
   });
 
+  it('should throws if max of inputs was exceeded', async () => {});
```

Comment:
Should you convert to draft whilst working on this test?

Reply:
Yes. Can you make this for me? I don't see the option for it.